SVGKit.podspec

```diff
@@ -2,7 +2,9 @@ Pod::Spec.new do |s|
   s.name        = 'SVGKit'
   s.version     = '2.0.1'
   s.license     = 'MIT'
-  s.platform    = :ios, '5.0'
+  s.osx.deployment_target = '10.10'
+  s.ios.deployment_target = '5.0'
```

Comment:
This is out of sync with the ios 8.0 min above?

Reply:
? This is framework target version, iOS 5 is from the SVGKit-iOS.xcproj target version. Mac OS X support is new created by me, not available before. So this does not obey any [sem version](https://semver.org/) constraint.

Reply:
If you think this Mac OS X min deployment target version is higher. (From our company's search, Mac OS X 10.10 and before contains only 10% total global Mac market). I can make it down to Mac OS X 10.9, using some compatible code.

Reply:
It was the iOS version I was worrying about, because I misunderstood which xcodeproj file I was looking at.

Good point re: Xcode blocking setting the deployment target lower. What I would like to do is get this huge patch approved, merged, ... and then declare 2.x complete, push a tagged version, and move to a 3.x branch, where we can throw away everything that's less than - say - iOS 9.